extensions-schema.json

```diff
@@ -25,7 +25,7 @@
         "description": "Relative path of the extension vsix file inside the git repo (i.e. when it is built by prepublish commands"
       },
       "custom": {
-        "type": "string",
+        "type": "array",
```

Comment:
I'd recommend you to allow both strings and arrays (if this wouldn't look too hacky):

```json
"custom": {
    "description": "Build using a custom script. Must output an `extension.vsix` file in the `location` directory.",
    "oneOf": [
        {
            "type": "string",
            "default": ""
        },
        {
            "type": "array",
            "default": []
        }
    ]
}
```

Reply:
I don't have a problem with adding this, but I am not sure if it is something that useful. If we stick to one format it looks to me like it does not have any disadvantages as opposed to an array, maybe just that a single command would be smaller in size. @manuth do you think that there could be any differences in the two approaches from how the commands are executed?

Reply:
No, not really - I think we'll be fine without this functionality!
I'm perfectly fine without it 😄